### PR TITLE
finalize 0.7.1

### PR DESCRIPTION
**Pull request by orbeckst**
- for #181 
- doc updates
- finalized CHANGES

### CODE REVIEW

**Review by ALescoulie — APPROVED** [review]
Good changes to documentation. Looking forward to dropping python 2 support in the near future.

File: doc/examples/README.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+========================
+ MDPOW Example: Benzene
+========================
+
+The ``examples`` directory contains input file for the solvation free
+energy calculation of benzene in solvent (water and octanol), using
+the OPLS-AA force field and the TIP4P water model.
+
+There are tow ways to use MDPOW:
```
Comment (iorga) [inline doc/examples/README.rst:11]: _tow_ should be _two_.

File: doc/examples/README.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  ITP mol    transferFE error          logPow err_logPow directory 
+  ---------- --------- ------ ------ -------- ---------- --------- 
+  BNZ         -8.71     1.94  logPow    +1.52     0.34   benzene
+
+(It also writes output to ``energies.txt``. See online docs.)
+
+Instead of running ``mdpow-solvationenergy`` for different solvents, one can
+also just run ``mdpow-pow DIRECTORY`` and it will run the separate free energy
+calculations automatically.
```
Comment (iorga) [inline doc/examples/README.rst:124]: I would put "automatically, and then compute the octanol-water partition coefficient."

**Review by iorga — APPROVED** [review]
I'm OK with all changes, but see my two very minor comments.

File: doc/examples/README.rst (hunk 1 above)
Comment (orbeckst) [reply doc/examples/README.rst]: thanks, 0124569845080456f40a02c718dacfe5ebd69c08

File: doc/examples/README.rst (hunk 2 above)
Comment (orbeckst) [reply doc/examples/README.rst]: added in 0124569845080456f40a02c718dacfe5ebd69c08

**Pull request merged**